packages/react/src/components/Avatar/Avatar.tsx

```diff
@@ -1,25 +1,33 @@
 import { Accessibility } from '@fluentui/accessibility'
+import {
+  getElementType,
+  getUnhandledProps,
+  useAccessibility,
+  useStyles,
+} from '@fluentui/react-bindings'
 import * as customPropTypes from '@fluentui/react-proptypes'
 import * as PropTypes from 'prop-types'
 import * as React from 'react'
+// @ts-ignore
+import { ThemeContext } from 'react-fela'
```

Comment:
Avatar.tsx should not refer to fela directly

Reply:
What do you think?

Reply:
It's not Fela specifics. We use this context only by historical reasons. We should use own context instead.